<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="userName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLabel" name="host">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>host</string>
   </property>
  </widget>
  <widget class="QLabel" name="database">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Uname_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Uhost_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Udatabase_line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="state_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>state</string>
   </property>
  </widget>
  <widget class="QLabel" name="Upassword">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Upassword_line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSee">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>table 조회</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Table Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnTable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>table 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="se_ta">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="show_table">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>741</width>
     <height>561</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Table Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnMkTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>table 제작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>840</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>데이터 삽입</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>261</width>
     <height>351</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="make_it">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>제작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>table 삭제</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="state_line_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>850</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>840</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>840</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdvance">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>advanced</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="state_line_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>190</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>To csv..</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
